galasa-ui/src/app/mysettings/page.tsx

```diff
@@ -65,6 +66,7 @@ export default async function MySettings() {
       />
       <TokenResponseModal refreshToken={refreshToken} clientId={clientId} onLoad={deleteCookies} />
       <DateTimeSettings />
+      <ResultsTablePageSizeSetting />
```

Comment:
If the user hasn't checked the box in the "Experimental Features" section to view test runs, I think this section should also be hidden.

Otherwise, it might confuse people to see a setting about test runs but they aren't able to see test runs 😅